### PR TITLE
Proper word wrap intents. fix for #11633

### LINKED ISSUE #11633: Confused word wrap
The word wrap in Brackets looks confused .

**Currently**
current_word_wrap fw

**Suggestion**
suggestion_word_wrap fw

### e.g. javascript code

**In Brackets** :(
b fw

**In Sublime Text** - Best to read, of course this is more visible in larger codes
s fw

I believe that the word wrap should respect indent code equal to Sublime Text and others.

What do you think?

### PR DESCRIPTION
**Pull request by abose**
fixes https://github.com/adobe/brackets/issues/11633
@petetnt @swmitra @wesleifreitas

### CODE REVIEW

File: src/editor/Editor.js [hunk 1]
```diff
@@ -982,6 +982,13 @@ define(function (require, exports, module) {
                 event.preventDefault();
             }
         });
+        // For word wrap. code adapted from https://codemirror.net/demo/indentwrap.html#
```
Comment (petetnt) [inline src/editor/Editor.js]: Nitpick: Capital C after period

**Comment by petetnt** [comment]
Tried it out against current master, confirmed to fix #11633. Implementation looks solid. LGTM

**Comment by abose** [comment]
Thanks @petetnt for reviewing.

**Comment by rroshan1** [comment]
Verified the fix. Changes look good. Merging.

**Pull request merged**